New autocomplete field type

johnsonpaul1014 opened the pull request:
This adds a new autocomplete field type capable of using local or remote sources.  It uses the jQuery UI Autocomplete widget.

jtsternberg commented:
Thanks!

jtsternberg commented:
Just a heads up, I cleaned up some whitespace issues, made the description easier to read and updated it to use the new metabox/field registration api: https://github.com/WebDevStudios/CMB2-Snippet-Library/commit/d3059fdd5093712258a1123b9e2b200b628685b0#diff-bbe65446904c9b5b958b4518ce8bb585

johnsonpaul1014 commented:
Sounds good.  Thanks for merging my changes in for the file type.  Now I can stay up to date.

Sent from my iPhone

> On Feb 15, 2015, at 7:23 PM, Justin Sternberg <notifications@github.com> wrote:
> 
> Just a heads up, I cleaned up some whitespace issues, made the description easier to read and updated it to use the new metabox/field registration api: d3059fd#diff-bbe65446904c9b5b958b4518ce8bb585
> 
> —
> Reply to this email directly or view it on GitHub.
>
